resources.go

```diff
@@ -1194,6 +1204,7 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_bot_channel_email":         {Tok: azureResource(azureBot, "ChannelEmail")},
 			"azurerm_bot_channel_slack":         {Tok: azureResource(azureBot, "ChannelSlack")},
 			"azurerm_bot_web_app":               {Tok: azureResource(azureBot, "WebApp")},
+			"azurerm_bot_channel_ms_teams":      {Tok: azureResource(azureBot, "ChannelMsTeams")},
```

Comment:
Shouldn't `MS` be capital? At least, it should in C# naming conventions. Alternatively, I'm fine with the name "ChannelTeams".

Reply:
I Like `ChannelTeams` better :) Moving to that!